bin/liquidator/src/rebalance.rs

```diff
@@ -565,4 +638,42 @@ impl Rebalancer {
 
         Ok(())
     }
+
+    fn is_lst(&self, mint: Pubkey) -> bool {
+        self.lst_mints.contains(&mint)
+    }
+
+    pub async fn init(&mut self, live_rpc_client: &RpcClient) {
+        if let Err(e) = self.load_lst_if_needed(live_rpc_client).await {
+            warn!("Could not load list of sanctum supported mint: {}", e);
+        }
+    }
+
+    async fn load_lst_if_needed(&mut self, live_rpc_client: &RpcClient) -> anyhow::Result<()> {
```

Comment:
nit: the `_if_needed` doesn't seem needed. Make a function that loads LST mints and returns them, then use it in init() instead of mutating `self` here?